config.schema.json

```
@@ -70,6 +76,15 @@
     }
   },
   "definitions": {
+    "proxy": {
+      "type": "object",
+      "properties": {
+        "path": { "type": "string"},
+        "url": { "type": "string"},
```

Do we have type `url`?

---

nope - but there is a format assertion for URIs
```suggestion
        "url": { "type": "string", "format": "uri"},
```